live-editing/configs/GridConfigGenerator.ts

```diff
@@ -338,6 +350,36 @@ export class GridConfigGenerator implements IConfigGenerator {
             })
         }));
 
+        configs.push(new Config({
+            component: GridSelectionTemplateExcelComponent,
+            additionalFiles: ["/src/app/grid/services/financialData.ts"],
+            appModuleConfig: new AppModuleConfig({
+                imports: [HttpClientModule, IgxAvatarModule, IgxBadgeModule, IgxButtonModule,
```

Comment:
Umm, any specific reason the Excel style sample that uses just the grid (header template is not even an icon) would need avatar, http, badge and so on.